### PR TITLE
Feature/footsteps dust new aproach

### PR DESCRIPTION
**Pull request by raulgonzalezupc**
New footsteps logic:
    - Now the footsteps starts to play when the player starts walking instead to emit each time that the player walks
    - Texture changed
    - Opacity over time
    - Color changed (visuals of the footsteps in general changed)

### CODE REVIEW

File: Scripts/src/Script/PlayerMovement.cpp [hunk 1]
```diff
@@ -187,7 +187,18 @@ void PlayerMovement::Move(int player, bool is_stunned)
 		is_jumping = false;
 		animation->SetBool("is_jumping", is_jumping);
 	}
-
+	if (!animation->IsOnState("Idle") && !animation->IsOnState("Dead") && !animation->IsOnState("Stunned") && !animation->IsOnState("Falling") && !animation->IsOnState("Jump"))
```
Comment (Dartemiss) [inline Scripts/src/Script/PlayerMovement.cpp:190]: So if player is attacking, or reviving particles will appear? Why dont put IsOnState("Run")?

**Review by Dartemiss — APPROVED** [review]

**Comment by yalania** [comment]
when you stop walking the footsteps dissapear, it feels a little bit strange, is there some way we can fix it?

File: Scripts/src/Script/PlayerMovement.cpp (hunk 1 above)
Comment (raulgonzalezupc) [reply Scripts/src/Script/PlayerMovement.cpp:190]: I thought that we also wanted it when attacking and dashing

**Comment by raulgonzalezupc** [comment]
> when you stop walking the footsteps dissapear, it feels a little bit strange, is there some way we can fix it?

I tried several times to fix it, but is super weird because I do the stop of the PS, and this should work, but it does not some times :/

**Review by yalania — APPROVED** [review]

**Pull request merged**